notebooks/samples/aihub/knn_classification/knn_classification.ipynb

```diff
@@ -0,0 +1,16305 @@
+{
```

Comment:
<p>Remove tf.contrib references</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>The <code>tf.contrib</code>  reference is in an HTML file that was generated after training. That HTML file is embedded into the notebook and shown as cell output. In order for me to remove that reference, we need to modify all of the ML components, do another release and rerun the training of all of the notebooks. </p><p> </p><p>This project may end at the end of the week and I am not sure if we will have the time to do that. Do you think we can merge them with the <code>tf.contrib</code>  for now?</p>

Reply:
Ok, that's fine.